server/src/middleware/errorHandler.ts

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { NextFunction, Request, Response } from 'express';
+import { ValidationError } from 'yup';
+
+export enum HttpCode {
```

Comment:
I think this enum is a great idea, but it could be moved under `types` instead and the 200 status could be included as well.

Reply:
moved under types and added few common http codes